verilog/analysis/checkers/forbid_negative_array_dim.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017-2020 The Verible Authors.
```

Comment:
might as well make this 2023 (other files too)

Reply:
Done! Updated also expression.cc and expression.h files